### PR TITLE
React: Fixed version detection

### PR DESCRIPTION
**Pull request by mxschmitt**
Example sites which are using React:
- Slack
- Dropbox
- Imgur
On all it works fine with PascalCase "React".

### CODE REVIEW

**Comment by gadcam** [comment]
@jvoisin should we keep both patterns ? (cf #2078)

**Comment by jvoisin** [comment]
Yes, since both are used. But we should remove the `React` one, and keep `React.version`.

File: src/apps.json [hunk 1]
```diff
@@ -7881,7 +7881,8 @@
       ],
       "js": {
         "React": "",
```
Comment (mxschmitt) [inline src/apps.json]: so remove this line right?

Comment (jvoisin) [reply src/apps.json]: Exactly :)

**Comment by gadcam** [comment]
@jvoisin My policy was to always keep both pattern : someone could easily remove the <code>React.version</code> object but not the <code>React</code> object.

**Comment by AliasIO** [comment]
Thanks!

**Pull request merged**